### PR TITLE
Added check for researching so athletics does not occur if so.

### PR DESCRIPTION
**Pull request by Dartellum**
Making the climbing section work as it should if research was started. Climb practice does not work with research. I added return if @researching to line 659.

### CODE REVIEW

File: crossing-training.lic [hunk 1]
```diff
@@ -63,9 +63,9 @@ class CrossingTraining
   def dump_junk
     return unless @settings.dump_junk
     return unless Time.now - @junk_timer > 300
-    return unless DRRoom.room_objs.size > @settings.dump_item_count
+    return unless DRRoom.room_objs.size > 13
 
-    bput('dump junk', 'You should just kick', 'You have marked this room', 'You cannot summon')
+    fput('dump junk')
```
Comment (Sarvatt) [inline crossing-training.lic]: You had unintentional changes in here and reverted the dump junk changes from earlier

Comment (Dartellum) [reply crossing-training.lic]: I had an earlier version then the one that was already pushed. I was in testing. :) Should I make the changes mentioned in the

Comment (Sarvatt) [reply crossing-training.lic]: yeah switch to the red ones and this whole hunk will go away :)

Comment (Dartellum) [reply crossing-training.lic]: Just did, please check. :)

**Comment by Sarvatt** [comment]
All good, thank ya!

**Pull request merged**